specifyweb/frontend/js_src/lib/components/Molecules/SvgIcon.tsx

```diff
@@ -239,6 +239,9 @@ const colorMapper = f.store<Partial<RR<keyof Tables, Gradient>>>(() => ({
   ...startsWith('DNA', colors.purple),
   ...startsWith('Sp', colors.lightBlue),
   ...startsWith('Workbench', colors.green),
+  ...startsWith('AbsoluteAge', colors.blue),
+  ...startsWith('RelativeAge', colors.lightBlue),
+  ...startsWith('TectonicUnit', colors.red),
```

Comment:
Going with these icon colors for the age tables for now to fix the unit test for this. Let me know if something else would be preferable